src/data-access/repositories/interfaces/device-repository.ts

```diff
@@ -1,8 +1,13 @@
-import { Device } from '../../../model/entities/device'
+import {Device} from '../../../model/entities/device'
 
 export interface DeviceRepository {
   getDevices(): Promise<Device[]>
+
   getDevice(deviceId: string): Promise<Device>
+
   save(device: Device): Promise<void>
+
   update(device: Device): Promise<void>
+
+  delete(deviceId: string): Promise<void>
```

Comment:
I think this might be your IDE settings, but it would be nice if it didn't add newlines to all our interfaces :)

Reply:
Hard to find this setting. Working on it!

Reply:
Does these settings match yours?

<img width="983" alt="Skærmbillede 2024-07-18 kl  08 52 20" src="https://github.com/user-attachments/assets/95779380-42ad-4bc9-9b45-0bb9d3baa656">

Reply:
Almost. Mine looks like this
![image](https://github.com/user-attachments/assets/219a6e75-74f6-44f1-a767-d372a971a134)